IdentityCore/tests/util/network/MSIDTestURLSession.h

```diff
@@ -51,4 +51,7 @@
 // Helper dispatch method that URLSessionTask can utilize
 - (void)dispatchIfNeed:(void (^)(void))block;
 
+// Required method to mock NSURLSession on iOS 13.
+- (void)defaultTaskGroup;
```

Comment:
just out of curiosity, what is this for?

Reply:
This method is being called internally by Apple when we create a network task in ios 13. The app crashes without it with "selector not found error".